sentry/sentry.py

```diff
@@ -46,22 +54,53 @@ def cog_unload(self):
         self.bot.remove_before_invoke_hook(self.before_invoke)
         return super().cog_unload()
 
+    @checks.mod()
```

Comment:
```suggestion
    @commands.guild_only()
    @checks.mod()
```
Checks applied to the command group apply to all subcommands of the group. Surely all commands in this command group would want to be guild only?
Unless the idea is that you'd only be allowed to use the test command outside of guilds, but that seems like odd functionality to me.

Reply:
Considering API keys not bot-level, rather than guild-level (correct my if I'm wrong), I think it's OK as-is.  
In my mind, pulling config (i.e. DSN) from a "global" scope essentially pulls this up to a global-level cog, just with some guild-level config (i.e. environment), so it makes sense to default-allow cog commands at a global level, restricting to guild-only where necessary.  
Bit of a ramble, hope I'm making sense 😄

Reply:
But wouldn't the error need to go to an environment, which is a guild-level config item? I'll openly admit my understanding of Sentry is primitive at best

Reply:
I don't think it _needs_ to, but my Sentry knowledge is also very limited. I'll defer to @BeryJu on this one.

Reply:
So environments are completely optional in Sentry, hence you can use the test command from anywhere. I also just noticed that this won't even work how I imagined it since there's only one global sentry client currently, so the environment would only be set when the cog is loaded, and then be ignored.